.github/workflows/valid-links.yml

```diff
@@ -1,6 +1,10 @@
 name: Check for dead links
 
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 3 * * *'
```

Comment:
I'm horrible at reading cron tabs. Does this mean once a day at three o'clock?

Reply:
Yes (same, but the workflow editor shows it in a tooltip :D)